libs/api-hjarntorget/lib/apiHjarntorget.ts

```diff
@@ -571,6 +572,22 @@ export class ApiHjarntorget extends EventEmitter implements Api {
     return statusChecker
   }
 
+  public async registerAbscense(child: EtjanstChild, startDate: Date, endDate: Date): Promise<void> {
```

Comment:
`getSchedule` takes `DateTime` as input parameters instead of `Date`. It would be good to be consistent with that :smile: